pims/image_sequence.py

```diff
@@ -73,19 +102,8 @@ class ImageSequence(FramesSequence):
     >>> frame_shape = video.frame_shape # Pixel dimensions of video
     """
     def __init__(self, path_spec, process_func=None, dtype=None,
```

Comment:
If this works:

    @classmethod
    def pkg_available(cls):
        return cls._imread is not None